types/party.d.ts

```diff
@@ -1,9 +1,11 @@
-import { Job, Role } from './job';
+import { Job } from './job';
 import { OutputStringsParamObject } from './trigger';
 
 export type BasePartyMemberParamObject = {
-  role?: Role;
+  role?: string;
```

Comment:
Why change the type of the `Role`? I think you should keep it remained but add `none` into `Role` instead.

Reply:
Because 'role' is not a fixed string after localization.

Reply:
Hmmm, in this case, I would recommend to add an extra field to store the localized name rather than changing the original one. What do you think?

Reply:
I agree with your idea. Initially, I also added a new attribute.

However, later I thought that existing users might have saved options with 'role' as the key. 

For instance, if I added a new one, let's say 'roleName,' how should we handle users who previously saved options with 'role'? 

Do we need to delete the 'role' option, and if not deleted, this option would only be effective for English language users, while 'roleName' would only be effective for non-English users. This would be quite awkward. 

I haven't come up with a good solution.